dbc2val/overlay.md

```diff
@@ -0,0 +1,23 @@
+# Use of overlay notes
+
+Generate a json file with annotations - like this if using VSS github repo
```

Comment:
no link to 'this' file

Reply:
This is still very much work in progress. I believe most things will be moved to the readme in this folder, and/or to another file referenced from the readme.